tavern/plugins.py

```diff
@@ -0,0 +1,154 @@
+"""VERY simple skeleton for plugin stuff
+
+This is here mainly to make MQTT easier, this will almost defintiely change
+significantly if/when a proper plugin system is implemented!
+"""
+import logging
+import requests
+
+from .util import exceptions
+from .request import RestRequest, MQTTRequest
+from .response import RestResponse, MQTTResponse
+
+logger = logging.getLogger(__name__)
+
+
+def get_extra_sessions(test_spec):
+    """Get extra 'sessions' for any extra test types
+
+    Args:
+        test_spec (dict): Spec for the test block
+
+    Returns:
+        dict: mapping of name: session. Session should be a context manager.
+    """
+
+    sessions = {}
+
+    if 1: # pylint: disable=using-constant-test
```

Comment:
Do you _have_ to load the HTTP stuff? presumably as of this branch, you could just be testing MQTT? If you do always need to load this, the `if 1` doesn't really make it easier to understand more than the comment.

Reply:
Fixed